README.md

```diff
@@ -261,11 +262,14 @@ If you use this library in academic settings,
 please cite the following paper.
 
 ```
-@unpublished{kanda2023engineering,
-    title={Engineering faster double-array Aho-Corasick automata},
-    author={Kanda, Shunsuke and Akabe, Koichi and Oda, Yusuke},
-    journal={Software: Practice and Experience},
-    note={to appear}
+@article{10.1002/spe.3190,
+    author = {Kanda, Shunsuke and Akabe, Koichi and Oda, Yusuke},
+    title = {Engineering faster double-array Aho–Corasick automata},
+    journal = {Software: Practice and Experience},
```

Comment:
```suggestion
    journal = {Software: Practice and Experience},
    year = {Early view}
```

> Should we modify the BibTeX fields after the next volume is published?

I think it is no problem to modify it at this time.

But,  `year` field is required for `@article`. "Early view" would be appropriate if there are no other ideas.

Reply:
Or, 2023 is also okay.